tests/integration/solidity_contracts/PlainOpcodes/test_plain_opcodes.py

```diff
@@ -188,6 +188,37 @@ async def test_should_create_counters(
             ).result.nonce
             assert nonce_final == nonce_initial + count
 
+        @pytest.mark.skip(
+            """
+            TODO: need to fix when there is no return data from the bytecode execution,
+            it calls CallHelper instead of CreateHelper when finalizing calling context
+            """
+        )
+        @pytest.mark.parametrize("bytecode", ["0x", "0x6000600155600160015500"])
+        async def test_should_create_contract_with_bytecode_and_no_return_data(
+            self,
+            plain_opcodes,
+            plain_opcodes_deployer,
+            bytecode,
+        ):
+            evm_addresses = await plain_opcodes.create(
+                bytecode=bytecode,
+                count=1,
+                caller_address=plain_opcodes_deployer,
+            )
+
+            # Assert that the contract was deployed with the correct bytecode.
+            assert len(evm_addresses) == 1
+            actual_bytecode = "".join(
+                [
+                    f"{x:02x}"
+                    for x in (
+                        await plain_opcodes.contract_account.bytecode().call()
+                    ).result.bytecode
+                ]
+            )
+            assert bytecode[2:] == actual_bytecode
```

Comment:
the test **deploys** a contract so
- it wouldn't be `plain_opcodes` holding the bytes
- the test says that it has no bytecode, so how come could we have the creation bytecode stored afterwards?

Reply:
Right it should have no bytecode because there is no return data, my bad will update this